website/docs/r/api_gateway_stage.html.markdown

```diff
@@ -129,6 +129,7 @@ In addition to all arguments above, the following attributes are exported:
 * `execution_arn` - The execution ARN to be used in [`lambda_permission`](/docs/providers/aws/r/lambda_permission.html)'s `source_arn`
   when allowing API Gateway to invoke a Lambda function,
   e.g. `arn:aws:execute-api:eu-west-2:123456789012:z4675bid1j/prod`
+  * `arn` - Amazon Resource Name (ARN)
```

Comment:
Nit: Looks like this is indented. Will fix on merge along with alphabetizing in the list.